web_app/testSite.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
+            <h1 id="new-task-form-title">Create new Task</h1>
+            <div id="new-task-form-body">
+                <br>
+                <div class="task-parameters">
+                    <div class="parameter-description">Task Name</div>   
+                    <div class="input-item">
+                        <input id="text-input" type="text"></input>
```

Comment:
Let's id this as `task-name-input` instead, and do that for the rest of the params